kanidm_tools/src/cli/session.rs

```diff
@@ -144,7 +144,8 @@ impl LoginOpt {
 
     fn do_totp(&self, client: &mut KanidmClient) -> Result<AuthResponse, ClientError> {
         let totp = loop {
-            println!("Enter TOTP: ");
+            print!("Enter TOTP: ");
+            io::stdout().flush().unwrap();
```

Comment:
Curious what the stdout flush is for, but I trust it's needed :)

Reply:
Without the flush the app just hangs because stdout's like "yeah, I buffered it, now what?" 🙄

Reply:
See I knew there would be a good reason :)

Reply:
Actually, can you add that as a comment above that line so that future archelogists know this too?

Reply:
Done :)